featherbb/View/admin/plugins.php

```diff
@@ -58,7 +59,7 @@
 <?php endforeach; ?>
                                 </tbody>
                             </table>
-                            <tfoot><strong>* <?php _e('Toggle active state'); ?></strong></tfoot>
+                            <tfoot><div id="toggle">* <?php _e('Toggle active state'); ?></div></tfoot>
```

Comment:
Could you place the tfoot tag one line above, just after the tbody of the table tag ? I placed it wrong.
And change the div id to "toggle-description". Yes, I'm a maniac :P